Packs/CreatePlbkDoc/Scripts/CreatePlbkDoc/CreatePlbkDoc.py

```diff
@@ -264,7 +264,7 @@ def main():
 
     while j < len(retVal):
 
-        PlaybookNameTempStr = retVal[j].get('id')
+        PlaybookNameTempStr = retVal[j].get('nameRaw')
```

Comment:
check what is the current XSOAR version (use the `is_demisto_version_ge` function):
if the version is below 6.6, use `id`.
otherwise, use `nameRaw`.